README.md

```diff
@@ -270,35 +147,9 @@ When file editing is done, you can click the button in the top right corner of t
 
 For more information about the `machete` file, look at the [reference blog post](https://medium.com/virtuslab/make-your-way-through-the-git-rebase-jungle-with-git-machete-e2ed4dbacd02).
 
+## Features List
 
-### Other actions
-
-On the left side bar you can find other actions (from top to bottom):
-- ![](docs/left_bar_actions/refresh.png) **Refresh Status** &mdash; refresh the graph displayed in main plugin window
-- ![](docs/left_bar_actions/toggle_listing_commits.png) **Toggle Listing Commits** &mdash; show or hide commits belonging to branches (for more details see `Toggle listing commits` section)
-- ![](docs/left_bar_actions/open_machete_file.png) **Open Machete File** &mdash; open the `machete` file in IntelliJ editor (to see what this editor can do see `Edit machete file` section)
-- ![](docs/left_bar_actions/fetch_all_remotes.png) **Fetch All Remotes** &mdash; equivalent to `git fetch --all` command
-- The most suitable actions (each is equivalent to one of the context menu actions) for a current branch that include one or more of:
-    - ![](docs/left_bar_actions/push.png) **Push Current Branch**
-    - ![](docs/left_bar_actions/pull.png) **Pull Current Branch**
-    - ![](docs/left_bar_actions/reset.png) **Reset Current Branch to Remote**
-    - ![](docs/left_bar_actions/slide_out.png) **Slide Out Current Branch**
-    - ![](docs/left_bar_actions/rebase.png) **Rebase Current Branch Onto Parent**
-    - ![](docs/left_bar_actions/override_forkpoint.png) **Override Fork Point of Current Branch**
-
-  Available action is selected based on a relation between a current branch and its parent and remote branch.
-- ![](docs/left_bar_actions/slide_in.png) **Slide In Branch Below Current Branch** &mdash; shortcut of [slide in](#slide-in-branch) action for current branch
-- ![](docs/left_bar_actions/help.png) **Show Help Window** &mdash; show window with a sample branch layout and explanation what parts of this graph mean
-
-
-### Multi-repository support
-
-Git Machete supports many git repositories in one project, including both regular repositories and submodules.
-If more than one repository is detected, the selection list will appear at the top of the main plugin window.
-In that case, each repository will have its own `machete` file.
-
-![](docs/multi_repo.gif)
-
+Please see the [features list](FEATURES-LIST.md) for more specific features description.
```

Comment:
I'd say `feature list`... generally I've observed that plural doesn't sound well in English in the "implicit genitive" (Polish: _lista ficzerów_ where _ficzerów_ is in genitive)

Reply:
Even Saxon genitive (`...'s` or `...'`) usually looks awkward when appended to a plural noun